test/specs/WebexMeeting/MeetingPage.js

```diff
@@ -0,0 +1,26 @@
+class MeetingPage {
+  get accessToken() { return $('#token'); }
+  get destination() { return $('#destination'); }
+  get saveTokenBtn() { return $('#md-button-1'); }
+  get displayWidgetBtn() { return $('#md-button-3'); }
+
+  get interstitialMeeting() { return $('.wxc-interstitial-meeting'); }
+
+  open() {
+    browser.url(process.env.WEBEX_TEST_PAGE_URL);
+
+    // override the local media api so that permission prompts don't show up
+    browser.execute('navigator.mediaDevices.getUserMedia = () => Promise.resolve(new MediaStream());');
+  }
+
+  loadWidget(accessToken, meetingDestination) {
```

Comment:
We will have other widgets in the future
```suggestion
  loadMeetingWidget(accessToken, meetingDestination) {
```